### PR TITLE
CONSOLE-2539: Add Chinese and Japanese translations

### PR DESCRIPTION
**Pull request by rebeccaalpert**
Adding translation files for zh-cn and ja-jp.

I made some modifications after doing a quick audit:
- Removed files associated with namespaces removed in namespace consolidation PR
- CamelCased the non-CamelCased resource keys in the public JSON files so the nav will be translated. (We sent the files for translation before the CamelCase PR merged. This led to much of the nav being in English because translations were not available for CamelCased keys.)
- Updated some bad English keys in the code, English JSON, Japanese JSON, and Chinese JSON:
    - Updated wont to won't
    - Updated numberic to numeric
    - Removed spacing before question mark

Chinese file issues I noticed and will bring up with Terry:
-  frontend/packages/console-app/locales/zh/console-app.json 
    - "Add HorizontalPodAutoscaler", Edit HorizontalPodAutoscaler", and "Remove HorizontalPodAutoscaler" were not translated
- frontend/public/locales/zh/public.json
    - "true" and "false" were not translated
    - "True" and "False" were not translated
    - "Service Accounts": "服务帐户", - was translated here but not in service-account.json or bindings.json
    - "Role Bindings": "角色绑定", - was translated here but not in role.json or bindings.json
- frontend/packages/pipelines-plugin/locales/zh/pipelines-plugin.json
    - PipelineRun inconsistent with translated Pipeline Run ("Pipeline Run": "管道运行”,)
- frontend/packages/console-shared/locales/zh/console-shared.json 
    - This \"Role\" is allowed to read the resource \"Pods\" in the core API group.": "这个 \"Role\" 可以读取核心 API 组中的资源\"Pods\"。", - Pod and Role inconsistent with other translations.
   - “This \"Role\" is allowed to read and write the \"Deployments\" in both the \"extensions\" and \"apps\" API groups.": "这个 
   \"Role\" 可以在 \"extensions\" 和 \"apps\" 组中读和写 \"Deployments\"。", - Role and Deployments inconsistent with other translations
    -  "This \"Role\" is allowed to read \"Pods\" and read/write \"Jobs\" resources in API groups.": "这个 \"Role\" 可以读 API 组中的 \"Pods\"，读/写 \"Jobs\" 资源。", - Role and Jobs inconsistent with other translations

Japanese file issues I noticed and will bring up with Terry:
- frontend/public/locales/ja/service-account.json
    - ServiceAccount is not translated even though it is in public.json
- frontend/public/locales/ja/role.json
    - RoleBinding is not translated even though it is in public.json
- frontend/public/locales/ja/public.json
    - "true" and "false", - not translated
    - "True" and "False" - not translated
- frontend/public/locales/ja/bindings.json
    - RoleBinding and ServiceAccount are not translated even though they are in public.json
- frontend/packages/operator-lifecycle-manager/locales/ja/olm.json
    - ServiceAccount is not translated even though it is in public.json
- frontend/packages/knative-plugin/locales/ja/knative-plugin.json 
    - "Knative Services": "Knative Serving", - I'm updating this in our code so we don't cause issues for users for now.
- frontend/packages/console-shared/locales/ja/console-shared.json 
    - "This \"Role\" is allowed to read and write the \"Deployments\" in both the \"extensions\" and \"apps\" API groups.": "この「ロール」は、「extensions」および「apps」API グループの両方で「Deployment」を読み取り、書き込むことができます。", - Deployment is not consistent with public.json
    - "This \"Role\" is allowed to read \"Pods\" and read/write \"Jobs\" resources in API groups.": "この「ロール」は「Pod」の読み取り、および API グループの「Job」リソースの読み取り/書き込みを許可します。", - Job is not consistent with public.json
    - "Allow reading a ConfigMap in a specific namespace (for RoleBinding)": "特定の namespace の ConfigMap の読み取りを許可（RoleBinding の場合)。", - RoleBinding is not consistent with public.json
    - "This \"Role\" is allowed to read a \"ConfigMap\" named \"my-config\" (must be bound with a \"RoleBinding\" to limit to a single \"ConfigMap\" in a single namespace).": "この「ロール」は「my-config」という名前の「ConfigMap」の読み取りを許可します（単一の namespace の単一の「ConfigMap」に制限するために「RoleBinding」でバインドされる必要があります)。", - RoleBinding is not consistent with public.json.

### CODE REVIEW

**Comment by spadgett** [comment]
/assign @sferich888 @ahardin-rh 
for PX and docs approval

We'll wait for QE approval until the PR is no longer WIP.

@rebeccaalpert Can you open a JIRA issue attached to the i18n epic to track? Thanks!

**Comment by spadgett** [comment]
```
frontend/packages/knative-plugin/locales/ja/knative-plugin.json 
    "Knative Services": "Knative Serving", - @spadgett are these different?
```

It looks like this is a mistake and should be `Knative Services` matching the key.

**Comment by rebeccaalpert** [comment]
@spadgett - I updated that manually. (See recent code.)

**Comment by andrewballantyne** [comment]
> ```
> frontend/packages/knative-plugin/locales/ja/knative-plugin.json 
>     "Knative Services": "Knative Serving", - @spadgett are these different?
> ```
> 
> It looks like this is a mistake and should be `Knative Services` matching the key.

Would appear to match the English value as well. 👍

**Comment by spadgett** [comment]
/assign @yapei 
I think we're ready for QE review.

File: frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json [hunk 1]
```diff
@@ -186,7 +186,7 @@
   "Node Table": "Node Table",
   "Create Storage Cluster": "Create Storage Cluster",
   "OCS runs as a cloud-native service for optimal integration with applications in need of storage and handles the scenes such as provisioning and management.": "OCS runs as a cloud-native service for optimal integration with applications in need of storage and handles the scenes such as provisioning and management.",
-  "nodesText": "The selected nodes will be labeled with <1>cluster.ocs.openshift.io/openshift-storage=\"\"</1> (unless they are already labeled). {{replica}} of the selected nodes will be used for initial deployment. The remaining nodes will be used by OpenShift as scheduling targets for OCS scaling.",
+  "nodesText": "The selected nodes will be labeled with&nbsp;<1>cluster.ocs.openshift.io/openshift-storage=&quot;&quot;</1>&nbsp;(unless they are already labeled). {{replica}} of the selected nodes will be used for initial deployment. The remaining nodes will be used by OpenShift as scheduling targets for OCS scaling.",
```
Comment (yanpzhan) [inline frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json:189]: &nbsp and &quot are still in string when language is English.
Screenshot from 2021-02-03 15-02-19

File: frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json [hunk 2]
```diff
@@ -252,7 +252,7 @@
...
-  "If you wish to use the Arbiter stretch cluster, a minimum of 4 nodes(2 different zones, 2 nodes per zone) and 1 additional zone with 1 node is required. All nodes must be pre-labeled with zones in order to be validated on cluster creation.": "If you wish to use the Arbiter stretch cluster, a minimum of 4 nodes(2 different zones, 2 nodes per zone) and 1 additional zone with 1 node is required. All nodes must be pre-labeled with zones in order to be validated on cluster creation.",
+  "If you wish to use the Arbiter stretch cluster, a minimum of 4 nodes (2 different zones, 2 zones per node) and 1 additional zone with 1 node is required. All nodes must be pre-labeled with zones in order to be validated on cluster creation.": "If you wish to use the Arbiter stretch cluster, a minimum of 4 nodes (2 different zones, 2 zones per node) and 1 additional zone with 1 node is required. All nodes must be pre-labeled with zones in order to be validated on cluster creation.",
```
Comment (yanpzhan) [inline frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json:255]: I'm doubted why "2 nodes per zone" are updated to "2 zones per node", in my understanding, a node only can be in one zone and a zone can contain many nodes.

File: frontend/packages/ceph-storage-plugin/locales/zh/ceph-storage-plugin.json [hunk 3]
```diff
@@ -220,8 +234,23 @@
...
+  "Provided": "提供",
+  "KMS service {{value}} already exist": "KMS 服务 {{value}} 已存在",
+  "An encryption key for each Persistent volume (block only) will be generated.": "将为每个持久性卷（仅限块）生成一个加密密钥。",
+  "Encrypted PVs cannot be cloned expanded or create snapshots.": "加密 PV 无法进行克隆、扩展或创建快照。",
+  "The last saved values will be updated": "最后保存的值将更新",
   "The Storage Class will be used to request storage from the underlying infrastructure to create the backing persistent volumes that will be used to provide the OpenShift Container Storage (OCS) service.": "存储类将用于从底层基础架构请求存储，以创建用于提供 OpenShift Container Storage(OCS)服务的后端持久性卷。",
   "The backing storage requested will be higher as it will factor in the requested capacity replica factor and fault tolerant costs associated with the requested capacity.": "请求的后端存储会更高，因为它会考虑请求的容量副本因素以及与请求的容量相关的容错成本。",
+  "If you wish to use the Arbiter stretch cluster, a minimum of 4 nodes (2 different zones, 2 zones per node) and 1 additional zone with 1 node is required. All nodes must be pre-labeled with zones in order to be validated on cluster creation.": "如果您想要使用 Arbiter 扩展集群，则需要至少有 4 个节点（2 个不同的区、每个节点有 2 个区）和 1 个带有 1 个节点的额外区。所有节点都必须预先标记区以便在创建集群时进行验证。",
```
Comment (yanpzhan) [inline frontend/packages/ceph-storage-plugin/locales/zh/ceph-storage-plugin.json:251]: Same doubt about "2 zones per node"

File: frontend/packages/ceph-storage-plugin/src/constants/ocs-install.ts [hunk 4]
```diff
@@ -22,7 +22,7 @@ export const requestedCapacityTooltip = (t: TFunction) =>
   );
 export const arbiterText = (t: TFunction) =>
   t(
-    'ceph-storage-plugin~If you wish to use the Arbiter stretch cluster, a minimum of 4 nodes(2 different zones, 2 nodes per zone) and 1 additional zone with 1 node is required. All nodes must be pre-labeled with zones in order to be validated on cluster creation.',
+    'ceph-storage-plugin~If you wish to use the Arbiter stretch cluster, a minimum of 4 nodes (2 different zones, 2 zones per node) and 1 additional zone with 1 node is required. All nodes must be pre-labeled with zones in order to be validated on cluster creation.',
```
Comment (yanpzhan) [inline frontend/packages/ceph-storage-plugin/src/constants/ocs-install.ts:25]: Same doubt about  "2 zones per node"

File: frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json (hunk 2 above)
Comment (rebeccaalpert) [reply frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json:255]: Thanks @yanpzhan. Tried to fix the spacing and messed it up. I'll fix it. :)

File: frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json (hunk 1 above)
Comment (rebeccaalpert) [reply frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json:189]: Thank you @yanpzhan!

**Comment by rebeccaalpert** [comment]
Addressed @yanpzhan's QE feedback.

**Comment by spadgett** [comment]
Hi, @yanpzhan. Can you review again as code freeze is tomorrow?

**Comment by spadgett** [comment]
Disregard, I see your approval. Thanks!

**Review by spadgett — APPROVED** [review]
/lgtm
/hold for PX approval from @sferich888

**Comment by spadgett** [comment]
> The label(s) `/label px-approve ` cannot be applied.

/label px-approved
/hold cancel

**Pull request merged**